### PR TITLE
Fix visible <br /> in shop address

### PR DESCRIPTION
**Pull request by SebBareyre**
Fix visible <br /> in shop address

### CODE REVIEW

File: views/templates/hook/payment_return.tpl [hunk 1]
```diff
@@ -29,7 +29,7 @@
 		{l s='Your check must include:' d='Modules.Checkpayment.Shop'}
 		<br /><br />- {l s='Payment amount.' d='Modules.Checkpayment.Shop'} <span class="price"><strong>{$total_to_pay}</strong></span>
 		<br /><br />- {l s='Payable to the order of' d='Modules.Checkpayment.Shop'} <strong>{if $checkName}{$checkName}{else}___________{/if}</strong>
-		<br /><br />- {l s='Mail to' d='Modules.Checkpayment.Shop'} <strong>{if $checkAddress}{$checkAddress}{else}___________{/if}</strong>
+		<br /><br />- {l s='Mail to' d='Modules.Checkpayment.Shop'} <strong>{if $checkAddress}<br />{$checkAddress nofilter}{else}___________{/if}</strong>
```
Comment (Quetzacoalt91) [inline views/templates/hook/payment_return.tpl]: Hmm, we never fond of not escaping the content displayed, for security reason.
Another solution coming in mind is using carriage return instead, and using nl2br methods.

Comment (Klemart3D) [reply views/templates/hook/payment_return.tpl]: So why `nofilter` is used into [_payment_infos.tpl_ file](https://github.com/PrestaShop/ps_checkpayment/blob/master/views/templates/front/payment_infos.tpl) ? It does not seem very coherent ...

Line 34 : `<dd>{$checkAddress nofilter}</dd>`

**Comment by Klemart3D** [comment]
And only for hook "PaymentReturn" (screen after ordered), the "<br />" tag isn't displayed into payment infos hook (last of order steps) or into confirmation mail.

File: views/templates/hook/payment_return.tpl (hunk 1 above)
Comment (Seb33300) [reply views/templates/hook/payment_return.tpl]: The `$checkAddress` value is defined by admin in Prestashop backoffice.
BTW, the `nofilter` tag is already applied to this value in others templates as indicated by @Klemart3D.

**Comment by Seb33300** [comment]
@PierreRambaud Can you please take a look on this.
It was reported 2 years ago and I am still encountering this issue today:
image
Thank you

**Comment by Progi1984** [comment]
Personnally, I prefer the approach in the PR #25

**Comment by PierreRambaud** [comment]
> @PierreRambaud Can you please take a look on this.
> It was reported 2 years ago and I am still encountering this issue today:
> image
> Thank you

Really sorry for the delay.
Like @Progi1984 I prefer the approach of #25 

I close this one in favor of #25 

Thanks everybody

**Pull request closed without merging**